hkm/views/views.py

```diff
@@ -1385,3 +1437,5 @@ def handler500(request):
     response = render_to_response('hkm/views/500.html', context)
     response.status_code = 500
     return response
+
+
```

Comment:
You can remove one of these newlines. :eyes:

Reply:
Removed extra empty lines.